workflowproj/workflowproj.go

```diff
@@ -38,6 +38,8 @@ import (
 	operatorapi "github.com/apache/incubator-kie-kogito-serverless-operator/api/v1alpha08"
 )
 
+// TODO: should we delete this whole file? seems used only the the associated test functions
```

Comment:
What do you mean? This is an external module that an actor can use to generate CRs based on SonataFlow projects. kn workflow uses it, for instance. 
But I'm curious to understand why you came to this conclusion. Maybe we need to communicate this better, as part of being part of the [README](https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/workflowproj/README.md).

Reply:
I was not aware of module this and I totally missed the `README`: now I understand that this is the tool used in `gen-manifest` utility. So, even if the functions in this file are not used in the operator, it must stay in the repo.

The reason why I bumped into this module is that it is performing some manipulation of the properties that now are no more needed: I will remove that part and also the generation of the `managed-props` ConfigMap that, I assume, will not be part of the `gen-manifest` output.
I will validate the manifests using the README, thanks for the pointer.

Please comment if you believe that the `gen-manifest` should not behave as I wrote above.

Reply:
Can you please just remove this line then?